src/tiledimage.js

```diff
@@ -751,6 +793,16 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     },
 
     /**
+     * @private
+     * Get the point around which this tiled image is rotated
```

Comment:
I prefer not to document private functions using doc comments (so they'll show up on the documentation pages). Having them documented here in the code is great, but we should just use normal // comments so they don't end up on the public documentation.